docs/resources/xray_watch.md

```diff
@@ -5,18 +5,98 @@ Provides a Xray watch resource.
 ## Example Usage
 
 ```hcl
-# Create a new Xray watch for all repositories
-resource "xray_watch" "example" {
-  name  = "watch-name"
-  description = "watching all repositories"
-  resources {
+# Create a new Xray watch for all repositories, assign policies
+resource "xray_watch" "all-repos" {
+  name        = "all-repos-watch"
+  description = "Watch for all repositories, matching the filter"
+  active      = true
+  watch_resource {
     type = "all-repos"
-    name = "All Repositories"
+    filter {
+      type  = "regex"
+      value = ".*"
+    }
   }
-  assigned_policies {
-    name = xray_policy.example.name
+  assigned_policy {
+    name = xray_security_policy.security1.name
+    type = "security"
+  }
+  assigned_policy {
+    name = xray_license_policy.license1.name
+    type = "license"
+  }
+  watch_recipients = ["test@email.com", "test1@email.com"]
+}
+```
+```hcl
+# Create a new Xray watch for a set of repositories, assign policies
+resource "xray_watch" "repository" {
+  name        = "repository-watch"
+  description = "Watch a single repo or a list of repositories"
+  active      = true
+
+  watch_resource {
+    type       = "repository"
+    bin_mgr_id = "default"
+    name       = "your-repository-name"
+    filter {
+      type  = "regex"
+      value = ".*"
+    }
+  }
+
+  watch_resource {
```

Comment:
So, is this the way to do plurality? same block repeated? Or, in an array?

Reply:
Yep.